prisma/schema.prisma

```diff
@@ -35,13 +37,28 @@ model Shift {
   startTime        DateTime
   endTime          DateTime
   workerSlots      Int
+  blockedWorkers   BlockedWorker[]
```

Comment:
- The blockedWorkers relation in the Shift model appears redundant and potentially problematic.It could lead to data inconsistency if a worker is blocked at facility level but not marked as blocked for specific shifts.
```suggestion
```

Reply:
i believe this might require, to find out for how many workers for blocked for a particular list and their info as well. for more futuristic usage and analytics